### DISCUSSION TITLE
How to change background colour per profile?

### DISCUSSION THREAD

**Discussion by tphillips-mu** [Eugeny/tabby#9795] [Q&A]
I'm trying out Tabby and my current use pattern I use different background colours for different profiles, to differentiate sessions to Prod, Test or Dev environment servers.

Is there any way to set a custom theme per Profile, or just a custom background colour override per Profile?

**Comment by Eugeny** [comment]
You can choose different color schemes per profile - see the Colors tab in the profile's settings

If you just want different background, you can clone and customize your current color scheme multiple times

**Reply by tphillips-mu** [reply]
Aha! I can see that option if I clone "ssh", but the Profiles I'm using are cloned off running "Git Bash" (since I run a script that dynamically changes ssh args for the destination), and those profiles do not have a Colors tab. 

Looking at the other Built-in profiles, some do have Colors (ssh, serial) and other's don't (telnet, raw socket, powershell, cmd, bash).

So more specifically, can a custom background be set for those profile types that don't support theming?
